### DISCUSSION TITLE
Is it possible to mirror a local container?

### DISCUSSION THREAD

**Discussion by ettoredn** [metalbear-co/mirrord#1657] [Q&A] [answered]
Let's say I have a docker image running PHP-FPM as its entrypoint. Would it be possible to mirror `docker exec <my-fpm-image>`?

If so, what about preventing outgoing xdebug connections from being routed from the pod, and instead route them to my local IDE running on `host-gateway`?

**Comment by aviramha** [comment, marked as answer]
Yes, it is possible. If you have outgoing xdebug connections, you can filter them out (and make it not go through the remote pod) by using #1607 that will soon be merged.
Right now you can have a Dockerfile that pulls in mirrord and your app image, and then runs the original command using mirrord.
We're considering adding support for more straightforward way - Can be tracked here. https://github.com/metalbear-co/mirrord/issues/1658
